pkg/execution/executor/executor.go

```diff
@@ -2168,24 +2218,35 @@ func (e *executor) handleGeneratorInvokeFunction(ctx context.Context, gen state.
 	opcode := gen.Op.String()
 	now := time.Now()
 
+	sid := telemetry.NewSpanID(ctx)
+	sidstr := sid.String()
+
 	// Always create an invocation event.
 	evt := event.NewInvocationEvent(event.NewInvocationEventOpts{
```

Comment:
We may also need to change a caller of this when we invoke via the API.

Reply:
what do you mean?

Reply:
We also create invocation events in the [REST API](https://github.com/inngest/inngest/blob/1e9c9f2d0deaf2be338d21627add65f07d1502d0/pkg/api/api.go#L248-L252) and the [GraphQL API](https://github.com/inngest/inngest/blob/main/pkg/coreapi/graph/resolvers/app_mutations.go); we'd need toa dd these details to those calls too.

Reply:
Nevermind - all of these keys are under `_inngest` anyway, which is a solid marker for "don't use this please."